### ISSUE TITLE
Missing zeroslash and zeroslash.alt

### ISSUE THREAD

**Issue by chrissimpkins** [googlefonts/googlesans-flex#299]
During the attempt to implement the `zero` feature code, the following glyphs were missing:

- zeroslash
- zeroslash.tf

```
fontmake.errors.FontmakeError: In 'regular/GoogleSansFlex.designspace': Generating fonts from Designspace failed: The following glyph names are referenced but are missing from the glyph set:
 zeroslash (first found at /Users/chris/code/google/googlesans-flex/sources/regular/family.fea:179:17)
 zeroslash.tf (first found at /Users/chris/code/google/googlesans-flex/sources/regular/family.fea:180:20)
```

@MariannaPaszkowska

**Comment by MariannaPaszkowska** [comment]
@chrissimpkins Those glyphs are in the sources:
[zero-slash.glif](https://github.com/googlefonts/googlesans-flex/blob/fb-wip/sources/roman/GoogleSansFlex-opsz6-wdth151-wght1000-ROND0.ufo/glyphs/zero-slash.glif)
[zeroslash.glif](https://github.com/googlefonts/googlesans-flex/blob/fb-wip/sources/roman/GoogleSansFlex-opsz6-wdth151-wght1000-ROND0.ufo/glyphs/zeroslash.glif)
[zeroslash.tf.glif](https://github.com/googlefonts/googlesans-flex/blob/fb-wip/sources/roman/GoogleSansFlex-opsz6-wdth151-wght1000-ROND0.ufo/glyphs/zeroslash.tf.glif)
They don't seem to import. I need to look into that.
@sannorozco what's the reason for duplicate glyphs zero-slash and zeroslash?

**Comment by chrissimpkins** [comment]
The glyphs will be available on the next import following the merge of https://github.com/googlefonts/googlesans-flex/pull/303

**Comment by chrissimpkins** [comment]
I think that we are ok to close?  Is there a duplicate glyph issue Marianna?

**Comment by sannorozco** [comment]
No duplicated glyph issue @chrissimpkins , it’s the middle bar composite used to make zeroslash, I talked with Marianna and we”re renaming it to avoid confusion.

**Comment by chrissimpkins** [comment]
Thanks!

**Comment by dberlow** [comment]
As you all know, I was traveling this day. 
 This rename broke the rule of not causing Google name changes to effect our design work.
   a.  I have test strings including all the names we put on the glyphs, all of which are final. I changed one  string for this, not a big deal.
   b. we use Robofont's "suffix" function to show opentype features in our spacing window via those glyph names having a "." separator and a common suffix.
   c. We do not and have never named parts with anything but a "-" between the name of the glyph 
       they are for, and the part that's being added.  
   d. If there are dramatic differences between the figure widths and the .tf width, I was going to need a zero-slash.tf. 

If you look at the sources since we started this font and at the sources of RobotoVF and RobotoFlex we have been fairly careful and consistent with this.
